crypto/kdf/sskdf.c

```diff
@@ -66,10 +66,16 @@ static const unsigned char kmac_custom_str[] = { 0x4B, 0x44, 0x46 };
 /*
  * Refer to https://csrc.nist.gov/publications/detail/sp/800-56c/rev-1/final
  * Section 4. One-Step Key Derivation using H(x) = hash(x)
+ * Note: X963 also uses this code with the only difference being that the
```

Comment:
Shouldn't this really be X9.63 here (i.e. with a dot ".")?

Reply:
Changed